Server/Utilities/SpringBoardAlerts.m

```diff
@@ -108,11 +108,6 @@ - (instancetype)init_private {
             [self loadLanguageIfExists:fixedPreferredLanguage:resultArray];
         }
 
-        // load "en" lang
-        if (![preferredLanguage hasPrefix:@"en"]) {
```

Comment:
Why is it removed? It was not only about "en-US"

Reply:
because it has only en-US need to be mapped. For any other en locales we have separate file with localizations which correctly called.